forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -164,24 +175,47 @@ internal class ConfigHelper: ConfigInterface {
      * @param context  The Application Context
      * @return FROptions from stored persistence
      */
+    @JvmOverloads
     override fun loadFromPreference(context: Context): FROptions {
-            val sharedPreferences: SharedPreferences =
-                context.getSharedPreferences(ORG_FORGEROCK_V_1_HOSTS, Context.MODE_PRIVATE)
-            return FROptionsBuilder.build {
-                server {
-                    url = sharedPreferences.getString(ConfigHelper.url, null) ?: ""
-                    realm = sharedPreferences.getString(ConfigHelper.realm, null) ?: ""
-                    cookieName = sharedPreferences.getString(ConfigHelper.cookieName, null) ?: ""
-                }
-                oauth {
-                    oauthClientId = sharedPreferences.getString(ConfigHelper.clientId, null)
-                    oauthScope = sharedPreferences.getString(ConfigHelper.scope, null)
-                    oauthRedirectUri = sharedPreferences.getString(ConfigHelper.redirectUri, null)
-                }
-                urlPath {
-                    endSessionEndpoint = sharedPreferences.getString(ConfigHelper.endSessionEndpoint, null)
-                    revokeEndpoint = sharedPreferences.getString(ConfigHelper.revokeEndpoint, null)
-                }
+        val sharedPreferences =
+            context.getSharedPreferences(ORG_FORGEROCK_V_1_HOSTS, Context.MODE_PRIVATE)
+
+        return FROptionsBuilder.build {
+            server {
+                url = sharedPreferences.getString(ConfigHelper.url, null) ?: ""
+                realm = sharedPreferences.getString(ConfigHelper.realm, null) ?: ""
+                cookieName = sharedPreferences.getString(ConfigHelper.cookieName, null) ?: ""
+            }
+            oauth {
+                oauthClientId = sharedPreferences.getString(ConfigHelper.clientId, null) ?: ""
+                oauthScope = sharedPreferences.getString(ConfigHelper.scope, null) ?: ""
+                oauthRedirectUri = sharedPreferences.getString(ConfigHelper.redirectUri, null) ?: ""
             }
+            urlPath {
+                endSessionEndpoint =
+                    sharedPreferences.getString(ConfigHelper.endSessionEndpoint, null)
+                revokeEndpoint = sharedPreferences.getString(ConfigHelper.revokeEndpoint, null)
+                logoutEndpoint = sharedPreferences.getString(ConfigHelper.logoutEndpoint, null)
+            }
+        }
     }
 }
+
+sealed class RevokeType
+object ClearToken: RevokeType()
+object Logout: RevokeType()
+
+/**
+ * TokenRemover Provide methods to clear the tokens when someone logout and to clear tokens.
+ */
+class TokenRemover(private val config: Config = Config(),
```

Comment:
Instead of creating a new TokenRemover class, I am thinking of adding a new static method under 
ConfigHelper.getPersistedConfig().tokenManager().revoke()

Reply:
we can talk about this , lets brainstorm